src/Packlist/PacklistPage.js

```diff
@@ -62,6 +76,12 @@ export default function PackListPage() {
   }
 }
 
+const EditImg = styled.img`
+  width: 40px;
+  height: auto;
+  margin: 7px 4px;
+`
+const EditButton = styled.button``
```

Comment:
You don't need the "EditButton" component if you don't add any styles.

Reply:
I forgot to delete it, after adding the image instead of text on the button.